rust/linux-client/src/main.rs

```diff
@@ -31,7 +31,6 @@ fn main() -> Result<()> {
         max_partition_time,
     )
     .unwrap();
-    tracing::info!("new_session");
```

Comment:
I liked this one because it showed that connlib returned without hanging. Does the "Firezone Started!" message have the same effect?

Reply:
> Does the "Firezone Started!" message have the same effect?

More or less. It does require successfully talking to the portal.

Reply:
We no longer have any locks in connlib so "hanging" would be very unusual. Also, panicking would display its own error message.

I've found this one to be redundant. The session constructor doesn't do anything other than spawn a big task so it doesn't really tell you anything I'd say.

Reply:
Okay. It definitely doesn't have to be `info` so if I ever need it I'll figure something out.